source/mainnet/smart-contracts/tutorials/wCCD/wCCD-interacting.rst

```diff
@@ -0,0 +1,752 @@
+.. _wCCD-interacting:
+
+========================================
+Interacting with the wCCD token protocol
+========================================
+
+.. note::
+
+    Before you start with part 2 of this tutorial, make sure you have access to a testnet node,
+    installed the ``concordium-client``, created an account in the mobile wallet app
+    that is funded with some CCD, and imported the keys
+    into the ``concordium-client`` configuration. If you haven't completed any of the above steps,
+    the piggy bank tutorial :ref:`part 3 <piggy-bank-preparing>`
+    will guide you through these setup steps.
+    If you haven't completed any of the above steps, you can continue
+    with :ref:`part 3 <wCCD-front-end-set-up>` of this tutorial also we recommend completing every part of this tutorial.
+
+Non-state-mutative functions
+----------------------------
+
+The protocol has four non-state-mutative functions (``balanceOf``, ``operatorOf``, ``tokenMetadata``, and ``supports``)
+that you can invoke on the ``proxy`` contract or the ``implementation`` contract.
+The schema is embedded into the ``implementation`` contract while you will need to provide a schema file
+when invoking the non-state-mutative functions on the ``proxy`` contract. All used schema files in this tutorial can be `downloaded <https://github.com/Concordium/concordium.github.io/tree/main/source/mainnet/smart-contracts/tutorials/wCCD/schemas>`_
+or you can create them as described in the
+comments of the `upgradable wCCD smart contract <https://github.com/Concordium/concordium-rust-smart-contracts/pull/128>`_.
+Because you will use a ``schema`` (embedded or provided with an extra flag),
+the input parameters can be provided with the ``--parameter-json`` flag.
+
+**TODO: create all the schemas for the state-mutative/non-state-muatative functions from the last version of the protocol that will be deployed so minor changes are included. Currently, only the state-mutative schemas are added to this PR.**
+
+.. _balanceCCD:
+
+Getting the CCD balance of an address
+=====================================
+
+.. dropdown:: Checking the CCD balance of an account (click here)
+
+    You can check the CCD balance of an account on `CCDScan <https://testnet.ccdscan.io/>`_.
+
+    .. image:: ./images/wCCD_tutorial_1.png
+        :width: 100 %
+
+.. dropdown:: Checking the CCD balance of a smart contract (click here)
+
+    You can check the CCD balance of a smart contract with this command:
+
+    .. code-block:: console
+
+        $./concordium-client contract show INDEX --grpc-port 10001
+
+    .. image:: ./images/wCCD_tutorial_3.png
+        :width: 100 %
+
+.. note::
+
+    The smallest unit of CCD is 1 micro CCD and equals the 10^{−6} (one millionth) of a CCD.
```

Comment:
`10^{−6}` is not displayed correctly in the rendered website (it is just shown as is).